workshop/exercise-7/README.md

```diff
@@ -138,6 +140,16 @@ spec:
 > Note that the annotations can also be used to gradually enable mTLS on individual services, 
 >
 
+## Quiz
+
+True or False?
+
+1. Istio Auth/Citadel provides each microservice with a strong, cryptographic, identity in the form of a certificate. The certificates' life cycle is fully managed by Istio.
+
+1. Istio provides microservices with mutually authenticated connections, without requiring application code changes.
```

Comment:
would be great if you could provide answer to this. :)

Reply:
Sure, will add [true / false] indication next to each, but then I'm not sure how this is a quiz if the answer is inlined next to the question ;-)